code-challenges.test.js

```diff
@@ -13,40 +13,111 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
+// b) Create the function that makes the test pass.
+// Explain your code:
+//created a test that would expect our 2 arrays to be returned mulitplied by 3 
+//when tested i got ReferenceError: multiplyBy is not defined so we got our good fail
+// then i created my function that would take in both arrays 
+// used two returns, one for each array, to map through and multiply our value and return each element multiplied by 3 to make our test pass. 
+// this took me a while even after watching the breakdown together. in the end i had put the wrong value in toequal for array two lol
+
+// describe("multiplyBy3", () => {
+//     it("return array multiplied by 3", () => {
+//         expect(multiplyBy3(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+//         expect(multiplyBy3(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+//       }) 
+// })   
+// const multiplyBy3 = (array, array2) => {
+//     return array.map((value) => value *3)
+//     return array2.map((value) => value *3)
+// }
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+
+
+
+    // const multiplyBy3 = (arrayOfNums, arrayOfNums2).map((value) => value *3)
+    //     return value
+    // const multiplyBy3 = (array1, array2) => {
+    //     array.map((value) => {
+    //      return value *3
+    //     })
+// const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray2 = [24, 27, 30, 33, 36]
```

Comment:
Louie, so good and so close!  Your test is great and your function logic and use of .map is so good.  Just one problem - you should only have one parameter.  Overall much improved.  Proud of you!